types/clusterize.js/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for clusterize.js 0.17
+// Project: https://github.com/NeXTs/Clusterize.js
+// Definitions by: Pr1st0n <https://github.com/Pr1st0n>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class Clusterize {
+    constructor(options: ClusterizeOptions);
+
+    destroy(clean?: boolean): void;
+    refresh(force?: boolean): void;
+    clear(): void;
+    getRowsAmount(): number;
+    getScrollProgress(): number;
+    update(data?: string[]): void;
+    append(rows: string[]): void;
+    prepend(rows: string[]): void;
+}
+
+interface ClusterizeOptions {
```

Comment:
I would put these in a `namespace Clusterize` so that they can be used as in
```ts
import Clusterize = require("clusterize");
const x: clusterize.Options = { ... };
```